src/rust/engine/src/nodes.rs

```diff
@@ -387,7 +387,7 @@ impl Select {
                   &[
                     externs::store_bytes(&result.0.stdout),
                     externs::store_bytes(&result.0.stderr),
-                    externs::store_i64(result.0.exit_code as i64),
+                    externs::store_i64(i64::from(result.0.exit_code)),
```

Comment:
Could this be `result.0.exit_code.into()`?

Reply:
It could, and now it is!